crates/experimental-api/tests/exe.rs

```diff
@@ -1,16 +1,17 @@
 use beerus_experimental_api::{
-    exe::exec,
+    exe::call,
     gen::{client::blocking::Client, FunctionCall},
 };
 
 mod common;
 
 #[test]
-fn test_exec() -> Result<(), common::Error> {
+fn test_call_deprecated_contract_class() -> Result<(), common::Error> {
     let Ok(url) = std::env::var("BEERUS_EXPERIMENTAL_TEST_STARKNET_URL") else {
         return Ok(());
     };
     let client = Client::new(&url);
+    tracing_subscriber::fmt::try_init().ok();
```

Comment:
Are you sure you want to initial the subscriber here?
The `.ok()` feels unnecessary.
(I didn't know there was a `try_init` function, that's nice!)

Reply:
This allows running tests individually and get logs, while also allows running tests in bulk without errors of the subscriber being already initialized.